docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -173,6 +173,23 @@ export default defineExtension({
 
 :::
 
+### Refactor of `routes`
+
+In this version we updated how `routes` parameter from `defineRemixExtension`
+was resolve. You can now pass directly `import.meta.url` and the routes from
+your `/routes` directory will automatically be resolved:
```

Comment:
```suggestion
In this version we updated how `routes` parameter from `defineRemixExtension`
is resolved. When using folder based routing, you should pass `import.meta.url` to the `routes` in your extension definition:
```

Reply:
Also add a note, if they added a placeholder `root.tsx` it can be removed, also check if there is any mention of it in the documentation which needs to be removed